src/lib/components/form.svelte

```diff
@@ -14,6 +14,10 @@
     let formValidated: boolean = false
     let formContainer: HTMLFormElement
 
+    let isPrivate = false
+    let isOpen = false
+    inputFields['isPrivate'] = isPrivate
```

Comment:
why don't initialize it like this above?

Reply:
@JuriBog this was not done, I mean instead of:
```
    let inputFields = {}
    let unsubscribe
    let formValidated: boolean = false
    let formContainer: HTMLFormElement
    let isPrivate = false
    let isOpen = false
    inputFields['isPrivate'] = isPrivate
```
    write
```
    let unsubscribe
    let formValidated: boolean = false
    let formContainer: HTMLFormElement
    let isPrivate = false
    let isOpen = false
    let inputFields = { isPrivate:isPrivate}
```